build.py

```diff
@@ -23,4 +28,98 @@ def decrypt():
 @task(_checkdir)
 def encrypt():
     '''Encrypt secrets.dec.tf file.'''
-    os.system("ansible-vault encrypt --output secrets.enc secrets.dec.tf && rm -rf secrets.dec.tf")
\ No newline at end of file
+    os.system("ansible-vault encrypt --output secrets.enc secrets.dec.tf && rm -rf secrets.dec.tf")
+
+@task(_checkdir)
+def layer_dependency(summary='False',quiet='False'):
+    """Check layer dependency from the current layers"""
+    try:
+        remote_states = _layer_dependency(path.get_working_path())
+    except Exception as e:
+        print('Error getting deps:',e)
+
+    if summary.lower() == 'true':
+        deps = {'this': []}
+        for rs in remote_states:
```

Comment:
if there was an exception (line 39), `remote_states` won't be defined so this will break

Reply:
fixed